### PR TITLE
Fix filename error

### PR DESCRIPTION
**Pull request by vincywindy**
If name has no ascii words，it will throw an error when starting game.
image

### CODE REVIEW

**Comment by vincywindy** [comment]
image

File: pyproject.toml [hunk 1]
```diff
@@ -50,6 +50,7 @@ Pillow = "10.3.0"
 numpy = "^1.26.4"
 fakeredis = "^2.21.3"
 yarl = "^1.9.4"
+django = "^5.0.6"
```
Comment (gantoine) [inline pyproject.toml:53]: This won't do, you'll need to find a way to do this without adding the entirety of django.

**Review by gantoine — CHANGES_REQUESTED** [review]
Also please re-open this against `master`, which is the branch we develop against. Anything merged into `release` will be overridden.

**Pull request closed without merging**